python/lsst/pipe/tasks/functors.py

```diff
@@ -73,18 +73,18 @@ class Functor(object):
     * `name`: A name appropriate for a figure axis label
     * `shortname`: A name appropriate for use as a dictionary key
 
-    On initialization, a `Functor` should declare what filter (`filt` kwarg)
+    On initialization, a `Functor` should declare what band (`filt` kwarg)
```

Comment:
In other places you renamed `filt` to `band`. Do you want to do the same here? It's a breaking change in both cases, mind you.

Reply:
Some of these functors also operate on non-coadds. I'm torn on this: I think the places I changed are the necessary ones, but I think not all of this is actually "band" (some of it is just "whatever filter name you want to specify, so long as it's consistent with the input data"). Maybe @yalsayyad has insight?

Reply:
I'm OK with leaving `filt` as a variable/kwarg name for a band name. The docstring  "Filter upon which to do the calculation" is a little ambiguous.  I don't really see the case where `filt` is specified on something other than a `deepCoadd_obj` working anyway, so it's prob extra work for flexibility that isn't there in the first place.